plugins/modules/mso_schema_site_anp_epg_domain.py

```diff
@@ -109,6 +109,47 @@
     description:
     - Distinguished name of EPG lagpolicy. This attribute can only be used with vmmDomain domain association.
     type: str
+  delimeter:
+    description:
+    - Which delimeter to use with this domain association. This attribute can only be used with vmmDomain domain association.
+    type: str
+  binding_type:
```

Comment:
I think we should add to the description that this is required in the version where it got introduced. Same applies if there are such constraints for the other attributes.

Reply:
For me it is about informing the user that it is a required attribute when on a specific version (4.2 I think but this would need to be verified) . The PR is fixing the issue: MSO Error 400: site-local EPG XX in site XX in template XX: BindingType is required for VMware VMM Domain. So I would say that a additional description would be good to have, since we are not enforcing in the required_if. @lhercot, @shrsr, @anvitha-jain, @sajagana what is your opinion on this?